src/plugins/WebOfTrust/util/BackgroundJobFactory.java

```diff
@@ -0,0 +1,50 @@
+package plugins.WebOfTrust.util;
+
+/**
+ * Factory for {@link BackgroundJob}s which allows all background jobs constructed using an instance
+ * of this class to be terminated with a single method invocation.
+ *
+ * @author bertm
```

Comment:
> even if the file is moved without Git being involved.

I recently looked at the Git repository of coreutils, and it is still visible who did the very first commit 20 years ago, when Git didn't even exist :)  If we ever migrate to another version control system again, it is highly likely that not losing history is made a requirement.
[Also, the guy still has the same mail address and still works on coretuils. Thats quite some dedication for a set of "boring" commandline tools :D]